R/derive_params_growth_height.R

```diff
@@ -0,0 +1,282 @@
+#' Derive Anthropometric indicators (Z-Scores/Percentiles-for-Height/Length)
+#' based on Standard Growth Charts
+#'
+#' Derive Anthropometric indicators (Z-Scores/Percentiles-for-Height/Length)
+#' based on Standard Growth Charts for Weight by Height/Length
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified in `sex`, `height`, `height_unit`, `parameter`, `analysis_var`
+#'   are expected to be in the dataset.
+#'
+#' @param sex Sex
+#'
+#'   A character vector is expected.
+#'
+#'   Expected Values: `M`, `F`
+#'
+#' @param height Current Height/length
+#'
+#'   A numeric vector is expected. Note that this is the actual height at the current visit.
+#'
+#' @param height_unit Height/Length Unit
+#
+#'   A character vector is expected.
+#'
+#'   Expected values: 'cm'
+#'
+#' @param meta_criteria Metadata dataset
+#'
+#'   A metadata dataset with the following expected variables:
+#'   `HEIGHT_LENGTH`, `HEIGHT_LENGTHU`, `SEX`, `L`, `M`, `S`
+#'
+#'   The dataset can be derived from WHO or user-defined datasets.
+#'   The WHO growth chart metadata datasets are available in the package and will
```

Comment:
Shall we list out the names of the metadata to be clear, so that the users do not need to search for the metadata(or say "see example for details")?

Reply:
This is a good idea, but I think we'll need to draw up a bigger vignette/article for metadata creation/preprocessing anyway